### DISCUSSION TITLE
Keyboard shortcut for file outline

### DISCUSSION THREAD

**Discussion by melissachang** [community/community#9648] [Other Feature Feedback, Questions, & Ideas]
It would be nice if we could see file outline - eg for Java file. Perhaps this could be another level in the hierarchy in left-hand-side:

![image](https://user-images.githubusercontent.com/10929390/148130027-681d46d8-5e40-45b9-b3d1-c1269be98fb7.png)

**Comment by look** [comment]
@melissachang just to clarify, by "file outline", do you mean a list of the classes, constants, methods, etc defined in the Java file like an IDE would show, or were you thinking of something else?

**Comment by melissachang** [comment]
Yes, what an IDE would show. Usually I just want methods; the others ones would be useful too.

**Comment by colinwm** [comment]
Great idea! I've been meaning to do this for a while, will implement this soon!
